ops/model.py

```diff
@@ -2555,7 +2555,7 @@ def _run(self, *args: str, return_output: bool = False,
 
             # pyright infers the first match when argument overloading/unpacking is used,
             # so this needs to be coerced into the right type
-            result = typing.cast('CompletedProcess[bytes]', result)
+            result = typing.cast('CompletedProcess[bytes]', result)  # type: ignore
```

Comment:
We shouldn't need a "type: ignore" comment as well as a cast. I think we can now drop this cast entirely (and the comment above it, and the related import of `CompletedProcess`).

Reply:
yep, the `type: ignore` is to actually supress a pyright error. I let the cast statement remain for the sake of explcitness. might just drop it now.